lib/semian/circuit_breaker.rb

```diff
@@ -2,11 +2,17 @@ module Semian
   class CircuitBreaker
     attr_reader :state
 
-    def initialize(exceptions:, success_threshold:, error_threshold:, error_timeout:)
+    def initialize(name, exceptions:, success_threshold:, error_threshold:, error_timeout:, permissions: 0660)
+      @name = "#{name}_cb"
       @success_count_threshold = success_threshold
       @error_count_threshold = error_threshold
       @error_timeout = error_timeout
       @exceptions = exceptions
+
+      @shared_circuit_breaker_data = ::Semian::CircuitBreakerSharedData.new(
```

Comment:
I'm not a fan of this naming. Why not just `@state` or `@shared_state`. Same with the class name.